pom.client.xml

```diff
@@ -645,6 +675,32 @@
           </reportSet>
         </reportSets>
       </plugin>
+
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <version>0.11.2</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
```

Comment:
@conniey is the revapi-java plugin dependency not needed here?

Reply:
No. I checked. It's not needed.